cypress/integration/data-import/view-all.spec.js

```diff
@@ -0,0 +1,63 @@
+import DataImportViewAllPage from '../../support/fragments/data_import/dataImportViewAllPage';
+import getRandomPostfix from '../../support/utils/stringTools';
+import fileManager from '../../support/utils/fileManager';
+
+describe('ui-data-import: "View all" log screen', () => {
+  // Create unique file name with given type to upload
+  const fileType = 'mrc';
+  const uniqueFileName = `test${getRandomPostfix()}.${fileType}`;
+
+  before(() => {
+    cy.login(
+      Cypress.env('diku_login'),
+      Cypress.env('diku_password')
+    );
+    cy.getToken('diku_admin', 'admin');
+
+    // create dynamically file with given name in fixtures
+    fileManager.createFile(`cypress/fixtures/${uniqueFileName}`);
+
+    // upload generated file
+    cy.uploadFile(uniqueFileName);
+    DataImportViewAllPage.gotoViewAllPage();
+
+    const query = '((status any "COMMITTED ERROR")) sortby completedDate/sort.descending';
+    cy.getJobLogs({ limit: 1, query });
+
+    // remove generated test file from fixtures after uploading
+    fileManager.deleteFile(`cypress/fixtures/${uniqueFileName}`);
+  });
+
+  it('C11112 allows to search with "Keyword"', () => {
```

Comment:
I think it's better to have 1-1 relation between test in test rail and 'it' function in TAF. We have some originally created old tests with the opposite approach, but it should be refactored.